Core/src/IfcGeometryConverter/FaceConverter.h

```diff
@@ -697,18 +696,15 @@ namespace OpenInfraPlatform {
 						//	IfcLoop <- IfcEdgeLoop, IfcPolyLoop, IfcVertexLoop
 						const EXPRESSReference<typename IfcEntityTypesT::IfcLoop>& loop = bound->Bound;
 						bool polyOrientation = bound->Orientation;
-						if (!loop) {
-							BLUE_LOG(warning) << "FaceConverter Problem with Face #" << faceID << ": IfcLoop #" << loop->getId() << " no valid loop.";
 
-							if (boundID == 0) 
-							{
-								break;
-							}
-							else 
-							{
-								continue;
-							}
+						if (!loop) {
+							throw oip::InconsistentGeometryException(face, " no valid loop.");
+						}
+						if (boundID == 0) 
+						{
+							break;
 						}

```

Comment:
Just delete this. It is handled by the exception already.

Reply:
Resolved:  d142297